sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -1935,6 +1935,25 @@ class MultiFileParquetPartitionReader(
 
   override final def getFileFormatShortName: String = "Parquet"
 
+  private var currentTargetBatchSize = targetBatchSizeBytes
+
+  override final def chunkedSplit(buffer: HostMemoryBuffer): Seq[HostMemoryBuffer] = {
+    if (!useChunkedReader) {
+      throw new SplitAndRetryOOM("GPU OutOfMemory: could not split inputs " +
+          "chunked parquet reader is configured off")
+    }
+    currentTargetBatchSize = currentTargetBatchSize / 2
+    if (currentTargetBatchSize < 10 * 1024 * 1024) { // 10 MiB
```

Comment:
Nit: Would be nice to name this constant in MiB and use it in the places where it's been copied along with formatting it in the error messages.

Reply:
also works well in connection with 

```Scala
org.apache.spark.network.util.ByteUnit.MiB.toBytes(10)
```